### PR TITLE
add cl-torrents

### PR DESCRIPTION
**Pull request by vindarel**
Hello,

so this is my stuff.
I'm ok to move it in a better category if you wish.

### CODE REVIEW

**Comment by azzamsa** [comment]
Thank you for such awesome app. The tutorial is very nice to read and explore.

I think it's better if we have `Torrent Client` section under `Browser`.

**Comment by vindarel** [comment]
(thanks! appreciated)

mmh "torrent client" ? A "torrent client" is Transmission or Qbittorrent so this doesn't seem the best section to me. What about just "Torrent" (or Torrent app) and under "Internet" ?

**Comment by azzamsa** [comment]
"Torrent" is fine :).

**Comment by vindarel** [comment]
Allright ! (you're uncorruptible^^)

done & rebased.

File: README.md [hunk 1]
```diff
@@ -236,12 +237,18 @@ For more, see [Awesome-cl](https://github.com/CodyReichert/awesome-cl).
...
 #### Browser
 
 - [nEXT](https://github.com/nEXT-Browser/nEXT) -  nEXT - The fastest productivity web-browser.
 
 See also [Closure](https://github.com/dym/closure) (using McClim) and [Lispkit](https://github.com/AeroNotix/lispkit) (using WebKit).
 
+#### Torrent
+
+- [cl-torrents](https://github.com/vindarel/cl-torrents)
+  - a library and app with a command line and readline interface to search for torrents on popular trackers.
```
Comment (azzamsa) [inline README.md]: I think you write the wrong format :).

See [https://github.com/azzamsa/awesome-cl-software#guidelines-to-contribute](https://github.com/azzamsa/awesome-cl-software#guidelines-to-contribute)

**Comment by azzamsa** [comment]
I think you have wrong format. see [https://github.com/azzamsa/awesome-cl-software#guidelines-to-contribute](https://github.com/azzamsa/awesome-cl-software#guidelines-to-contribute)

**Pull request merged**